pkgs/by-name/al/albyhub/package.nix

```
@@ -0,0 +1,113 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  fixup-yarn-lock,
+  nodejs,
+  pkgs,
+  yarn,
+  stdenv,
+  makeWrapper,
+  callPackage,
+  go,
+}:
+
+let
+  ldkNode = callPackage ./ldk-node { };
+  ldkNodeGo = callPackage ./ldk-node-go {
+    inherit ldkNode;
+  };
+
+in
+
+buildGoModule rec {
+  pname = "albyhub";
+  version = "v1.17.1";
+
+  src = fetchFromGitHub {
+    owner = "getAlby";
+    repo = "hub";
+    rev = "v1.17.1";
```

```suggestion
    tag = "v{version}";
```
`tag` ensures this doesn't accidentally fetch a branch